### ISSUE TITLE
Subs. Error - Please Refresh the Page 

### ISSUE THREAD

**Issue by Kuroo2021** [Voldrix/onlyfans-dl-2#92]
![Screenshot 2023-07-06 213738](https://github.com/Voldrix/onlyfans-dl-2/assets/18900059/3941d616-4e25-4e20-9d16-36cd26f421d1)

After reapplying the information for my OF, I get this error and not sure what the issue may be.

**Comment by insomniacdev212** [comment]
real issue is the request is bad if you try to do a single creator you get 

```
USER_ID auth failed
http 400

Update your browser user-agent variable, then sign back in to OF and update your session variables.
https://ipchicken.com/
```

**Comment by insomniacdev212** [comment]
seems like OF has modified their request structure.

**Comment by Voldrix** [comment]
Fixed with the latest commit, see #90 for details.
